use advent_of_code_2019::day_10;

use std::error::Error;

fn main() {
    println!("Day 10 result: {:?}", day_10::day_10());
}
